### ISSUE TITLE
Embed codepages in .Core assembly

### ISSUE THREAD

**Issue by ikkentim** [ikkentim/SampSharp#283]

**Comment by ikkentim** [comment]
serializer w/ verify
```
            foreach (var file in Directory.GetFiles(@"sampsharp\env\codepages", "*.txt"))
            {
                if(file.Contains("README"))
                    continue;
                
                var cp = CodePageEncoding.Load(file);

                var datFile = Path.ChangeExtension(file, ".dat");
                if (File.Exists(datFile))
                    File.Delete(datFile);

                using (var datStr = File.OpenWrite(datFile))
                    CodePageEncoding.Serialize(cp, datStr);

                // verify
                using (var datStr = File.OpenRead(datFile))
                {
                    var cpRd = CodePageEncoding.Deserialize(datStr);

                    if (cp.ConversionTable.Count != cpRd.ConversionTable.Count)
                    {
                        Console.WriteLine($"count mismatch for {file}: {cp.ConversionTable.Count} != {cpRd.ConversionTable.Count}");
                        return;
                    }
                    
                    var srcArr = cp.ConversionTable.ToArray();
                    var rdArr = cpRd.ConversionTable.ToArray();

                    for (var i = 0; i < srcArr.Length; i++)
                    {
                        if (srcArr[i].Key != rdArr[i].Key)
                        {
                            Console.WriteLine($"key error at {file}:{i}: {srcArr[i].Key} != {rdArr[i].Key}");
                        }
                        if (srcArr[i].Value != rdArr[i].Value)
                        {
                            Console.WriteLine($"value error at {file}:{i}: {srcArr[i].Value} != {rdArr[i].Value}");
                        }
                    }

                    Console.WriteLine("OK " + file);
                }
            }

            return;
```

**Comment by ikkentim** [comment]
Closed by a312d98768ecc2a69ab7a7af9ce4eb67b4458492
